### ISSUE TITLE
Implement special format for highstate output

### ISSUE THREAD

**Issue by martinhoefling** [martinhoefling/molten#3]

**Comment by bbinet** [comment]
+1
something similar to Saltpad highstate format would be great ![Saltpad highstate format](https://github.com/tinyclues/saltpad/raw/master/screenshots/highstate_result.png):

**Comment by martinhoefling** [comment]
Yes, so the current structured output is not very helpful - I agree. The plan is to switch to react-json-tree or something similar. Rendering a highstate like this would require some preprocessing though. So I'll leave open if I'll do that in the next version or afterwards.
